main.js

```diff
@@ -0,0 +1,128 @@
+                <ul class="flex-cards">
+                  <li class="all-list first">${project.subtitle}</li>
+                  <li class="all-list second">${project.bullet}</li>
+                  <li class="all-list second">${project.backEnd}</li>
+                  <li class="all-list second">${project.bullet}</li>
+                  <li class="all-list second">${project.year}</li>
+                </ul>
+                <p class="p3">${project.shortDescription}</p>
+                <ul class="flex-button tech_lang">
+                ${project.languages.map((tech) => `<li class="button">${tech}</li>`).join('')}
+                </ul>
+                <br>
+                <input class="btns see_btn" type="button" value="See Project">
+            </div>
+        </div>
+`);
+
+const modalContainer = document.querySelector('.modal');
+const recentWork = document.querySelector('.myWork');
+const parser = new DOMParser();
+projectCards.forEach((projectCard, index) => {
+  const projectIndex = parser.parseFromString(projectCard, 'text/html').body.firstChild;
+  const popupModal = `
+<div class="project-popup-hidden ">
+  <div class="backgroundpop"></div>
+    <div class="project-popup">
+      <div class="project-header">
+        <input type="button" value="&#10006" class="fa-solid fa-xmark project-close close">
+        <h2>${projects[index].name}</h2>
+        <p class="all-list first">${projects[index].subtitle} <span class="all-list second bullet">${projects[index].bullet}</span><span class="all-list second">${projects[index].backEnd}</span><span class="all-list second bullet">${projects[index].bullet}</span> <span class="all-list second">${projects[index].year}</span></p>
+
+        <div class="card_img card1">
+          <img class="imagedesktop project-img"  src="${projects[index].image}" alt="card image">            
+          <div class="project-footer">
+          <p class="project-desc">${projects[index].description}</p>
+        <div>
+          <ul class = ("tech_lang")>
+            ${projects[index].technologies.map((tech) => `<li>${tech}</li>`).join('')}
+          </ul>
+
+          <div class="project-links">
+            <a href="${projects[index].seeLive}">See Live <img src="./assets/images/seeLive.svg" alt="see live icon"> <i class="close"></i></a>
+            <a href="${projects[index].seeSource}">See Source <img src="./assets/images/gitIcon.svg" alt="github icon"> <i class="close"></i></a>
+          </div>
+        </div>
+      </div>
+    </div>
+</div>
+`;
+
+  const popModal = parser.parseFromString(popupModal, 'text/html').body.firstChild;
+  const seeProject = projectIndex.querySelector('.see_btn');
+  const closeBtn = popModal.querySelector('.close');
+  seeProject.addEventListener('click', () => {
+    popModal.classList.toggle('project-popup-hidden');
+  });
+  closeBtn.addEventListener('click', () => {
+    popModal.classList.toggle('project-popup-hidden');
+  });
```

Comment:
- [ ] As suggested by the previous code reviewer. I kindly suggest that you work on making the image take it's maximum width like it does for desktop. I think the best solution is to use the bigger image i.e desktop images for your projects. You can download them from the Figma design

| Your Design  | Figma Design  | 
| ------------- |-------------:  |
| <img width="219" alt="image" src="https://user-images.githubusercontent.com/52934128/190845923-af3e7a9b-664e-4889-b461-b41fb2b5c340.png">     | ![190701945-a054441c-e2ba-4ddb-afb7-579b2802f4f8](https://user-images.githubusercontent.com/52934128/190845931-5a179ce4-92d3-41c9-81e2-3bc82cc3924b.png)   |